### PR TITLE
Update DBProfileDelegate.cs

### PR DESCRIPTION
**Pull request by tiagograf-AOT**
## Status
**Ready**

## Fixes or Implements [AB#8108](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/8108)
- [ ] Enhancement
- [x] Bug
- [ ] Documentation

## Description:
Fixes the timezone issue when getting number of logged in users.

### CODE REVIEW

File: Apps/Database/src/Delegates/DBProfileDelegate.cs [hunk 1]
```diff
@@ -160,10 +160,10 @@ public int GeUnregisteredInvitedUsersCount()
         /// <inheritdoc />
         public int GetTodayLoggedInUsersCount(TimeZoneInfo timezone)
         {
-            DateTime startDate = TimeZoneInfo.ConvertTimeFromUtc(DateTime.UtcNow.Date, timezone);
+            DateTime startDate = TimeZoneInfo.ConvertTimeFromUtc(DateTime.Today.ToUniversalTime(), timezone);
             int result = this.dbContext.UserProfile
                 .Count(u => u.LastLoginDateTime.HasValue &&
-                TimeZoneInfo.ConvertTimeFromUtc(u.LastLoginDateTime.Value, timezone) >= startDate);
+               u.LastLoginDateTime.Value >= startDate.ToUniversalTime());
```
Comment (FuriousLlama) [inline Apps/Database/src/Delegates/DBProfileDelegate.cs]: what is the point of converting from utc, to convert back to utc here? Also, the internal date needs to be in the local timezone

**Comment by healthgatewaybc** [comment]
Medication Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_412)

File: Apps/Database/src/Delegates/DBProfileDelegate.cs (hunk 1 above)
Comment (tiagograf-AOT) [reply Apps/Database/src/Delegates/DBProfileDelegate.cs]: see discussion in rocket chat

**Comment by tiagograf-AOT** [comment]
How it works:

Today Local time gets converted to UTC
Then it gets converted to Vancouver tmz
When doing the query (since records are in utc) i convert the vancouver today date to utc (instead of converting all records to vancouver tmz which was not supported by linq)

**Comment by healthgatewaybc** [comment]
Medication Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_412)

**Comment by healthgatewaybc** [comment]
Medication Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_412)

**Pull request merged**